content/ew-issue-z201-[2017-04-07].yaml

```diff
@@ -0,0 +1,239 @@
+        image: "https://github.com/ember-weekly/ember-weekly/raw/master/assets/sponsors/simplabs-gif.gif"
+        advert: true
+        headlines:
+            -
+                title: "Pro Ember.js: An advanced 2-day training event in Berlin"
+                description: "We'll tackle progressive web apps, modular architecture patterns, performance tuning, authentication & more!"
+                advert: true
+                link: http://ember-workshop.simplabs.com
+                domain: simplabs.com
+
+
+    -
+        heading: "Reading"
+        headlines:
+            -
+                title: "Testing Ember Component Actions When Components Build On Top Of Components"
+                description: ""
+                link: https://www.mutuallyhuman.com/blog/2017/03/17/testing-ember-component-actions-when-components-build-on-top-of-components
+            -
+                title: "Authentication with Ember.js and Passport"
+                description: ""
+                link: https://medium.com/@duizendnegen/authentication-with-ember-js-and-passport-2b5767a74703
+            -
+                title: "Multiple Yield Blocks with Contextual Components in Ember.js"
+                description: ""
+                link: http://yoranbrondsema.com/multiple-yield-blocks-contextual-components-ember-js/
+
+
+
+    -
+        heading: "Code"
+        headlines:
+            -
+                title: "ember-modal-dialog: Beggar's Point (v1.0.0)"
+                description: ""
+                link: https://github.com/yapplabs/ember-modal-dialog/releases/tag/v1.0.0
+            -
+                title: "ember-modal-dialog: Rags to Riches (v2.0.0-beta.0)"
+                description: ""
+                link: https://github.com/yapplabs/ember-modal-dialog/releases/tag/v2.0.0-beta.0
+            -
+                title: "Vim + Ember.js: vim-emberlayout"
+                description: ""
+                link: https://github.com/sukima/vim-emberlayout
+            -
+                title: "ember-web-workers"
+                description: ""
+                link: https://github.com/BBVAEngineering/ember-web-workers
+            -
+                title: "Gemini 2 - fast and safe way to clean up duplicate files on mac"
+                description: "Cut the chaos and free up enormous disk space by safely, and automatically mass removing duplicate files"
+                link: http://go.hackingui.com/macpawember100417
+                domain: macpaw.com
+                advert: true
+
+
+    # -
+    #     heading: "Jobs"
+    #     advert: true
+    #     headlines:
+    #         -
+    #            title: ""
+    #            description: ""
+    #            link:
+
+
+    -
+        heading: "Watch/Listen"
+        headlines:
+            -
+                title: "Getting Started With Glimmer.js A Quick First Look"
+                description: ""
+                link: https://www.youtube.com/watch?v=IaW3rlE42I0
```

Comment:
1 more to add here (although I haven't checked if it's already been used in another issue as it's an older video):

```yaml
            -
                 title: "Introduction to Ember Power Calendar"
                 description: ""
                 link: https://pusher.com/sessions/meetup/ember-london/introduction-to-ember-power-calendar